spec/rails_helper.rb

```diff
@@ -58,4 +58,12 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
```

Comment:
This sort of thing should go in a file in `spec/support/` as it makes it easier to upgrade rspec-rails later without worrying too much about incorporating changes to the `rails_helper` that new versions of rspec-rails would bring.